src/node_process_methods.cc

```diff
@@ -207,6 +207,13 @@ static void MemoryUsage(const FunctionCallbackInfo<Value>& args) {
           : static_cast<double>(array_buffer_allocator->total_mem_usage());
 }
 
+static void GetConstrainedMemory(const FunctionCallbackInfo<Value>& args) {
+  double value = static_cast<double>(uv_get_constrained_memory());
+  if (value) {
+    args.GetReturnValue().Set(value);
+  }
```

Comment:
Optional nit: it's a good idea to restrict the use of semantically wrong types (`double`) to the smallest possible scope.

```suggestion
  uint64_t value = uv_get_constrained_memory();
  if (value != 0) {
    args.GetReturnValue().Set(static_cast<double>(value));
  }
```

Reply:
Fixed.